crt_portal/cts_forms/views.py

```diff
@@ -1232,7 +1234,8 @@ def get(self, request, id=None):
         shared_report_fields['assigned_section'] = first_report.assigned_section
         shared_report_fields['status'] = first_report.status
         shared_report_fields['retention_schedule'] = first_report.retention_schedule
-        form = BatchReviewForm(user=request.user, instance=batch)
+        can_review_batch = request.user.has_perm('cts_forms.review_dispositionbatch') if request.user else False
```

Comment:
interesting - in what case are we making requests to this route without a user set?